jumanji/environments/routing/mmst/viewer_test.py

```diff
@@ -42,23 +39,9 @@ def test__animation(deterministic_mmst_env: Tuple[MMST, State, TimeStep]) -> Non
     viewer = MMSTViewer(
         env.num_agents,
     )
-    step_fn = jax.jit(env.step)
     save_path = "render_test_gif.gif"
 
-    states = [state]
-    actions = [
-        jnp.array([0, 4]),
-        jnp.array([3, 5]),
-        jnp.array([7, 4]),
-        jnp.array([6, 8]),
-        jnp.array([10, 10]),
-        jnp.array([9, 4]),
-    ]
-
-    for action in actions:
-        state, _ = step_fn(state, action)
-        states.append(state)
-
+    states = 2 * [state]
```

Comment:
Why `2 *`?

Reply:
This is testing the animation. So we just want to pass a list of states and see if the animation code runs. It doesn't really matter if the states are different or not.

Reply:
The test was taking ages and does not require all previously generated states to be rendered, I believe only 2 states are sufficient to test `animate`.

Reply:
Understood 👍